app/assets/javascripts/controllers/cloud_topology/cloud_topology_controller.js

```diff
@@ -0,0 +1,278 @@
+/* global miqHttpInject */
+
+miqHttpInject(angular.module('cloudTopologyApp', ['kubernetesUI', 'ui.bootstrap', 'ManageIQ']))
+.controller('cloudTopologyController', CloudTopologyCtrl);
+
+CloudTopologyCtrl.$inject = ['$scope', '$http', '$interval', '$location', 'topologyService'];
+
+function CloudTopologyCtrl($scope, $http, $interval, $location, topologyService) {
```

Comment:
@himdel so the container, network and container topology controller are basically the same. What are our options to extract it to some base controller? We should be able to create a base topology controller, right?